Whiskey/Tasks/Invoke-WhiskeyDotNetBuild.ps1

```diff
@@ -32,9 +32,10 @@ function Invoke-WhiskeyDotNetBuild
     }
 
     $outputDirectory = $TaskParameter['OutputDirectory']
-    if ($outputDirectory)
+    if ($outputDirectory -and ([IO.Path]::IsPathRooted($outputDirectory)))

```

Comment:
What happens if you pass an absolute path to this? Does it handle it? Can we just pass it as-is? Do we need to validate?

Reply:
Absolute path works fine. I was just following our design pattern where we generally don't allow absolute paths in Whiskey task properties.